
Fix from gcc CVS for the bug:
  eabi.S:255: Error: operand out of range (6 is not between 0 and 1)

  - Rene Rebe <rene@exactcode.de>

RCS file: /cvs/gcc/gcc/gcc/config/rs6000/eabi.asm,v
retrieving revision 1.11
retrieving revision 1.11.32.1
diff -u -r1.11 -r1.11.32.1
--- gcc/gcc/config/rs6000/eabi.asm	2002/09/20 23:46:58	1.11
+++ gcc/gcc/config/rs6000/eabi.asm	2004/09/30 03:52:59	1.11.32.1
@@ -252,7 +252,7 @@
 
 .Lcvt:
 	lwzu	6,4(3)				/* pointer to convert */
-	cmpi	0,6,0
+	cmpwi	0,6,0
 	beq-	.Lcvt2				/* if pointer is null, don't convert */
 
         add	6,6,12				/* convert pointer */
